_maps/map_files/ChimeraStation/ChimeraStation.dmm

```diff
@@ -33159,7 +33184,12 @@
 /obj/structure/window/reinforced{
 	dir = 1
 	},
-/mob/living/simple_animal/hostile/retaliate/terry,
+/obj/structure/bed/dogbed/mcgriff,
```

Comment:
Very funny, sadly for you I read code now

Reply:
is not mcgriff, is called mcgrieff, diferrent dog

Reply:
Changes still requested,